plugins/CoreAdminHome/templates/home.twig

```diff
@@ -36,10 +36,10 @@
         {{ feedbackHelp|raw }}
     {% endif %}
 
-    {% if hasPremiumFeatures and isMarketplaceEnabled %}
+    {% if hasPremiumFeatures and isMarketplaceEnabled and isInternetEnabled %}
         <div piwik-widget-loader='{"module":"Marketplace","action":"getPremiumFeatures"}'></div>
     {% endif %}
-    {% if hasNewPlugins and isMarketplaceEnabled %}
+    {% if hasNewPlugins and isMarketplaceEnabled and isInternetEnabled %}
```

Comment:
Could you add the check for the RssWidget some lines below as well? It also requires connection to the internet.